spec/spec_helper.rb

```diff
@@ -6,6 +6,7 @@
 require 'tempfile'
 require 'pry'
 require 'thread'
+require 'pathname'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.